### ISSUE TITLE
Default `--platform` incorrectly determined on MacOS

### ISSUE THREAD

**Issue by devklick** [devklick/gdman#25]
When running a command such as `gdman install` or `gdman uninstall` on MacOS, if you do not include the `--platform | -p` argument, it's incorrectly defaulted to `mac-os`. However, this arg only accepts `macos`, `windows`, `linux`. 

This seems to be a problem with how the enum variants are derived. Possible culprit code:
https://github.com/devklick/gdman-rs/blob/a3d3ac0e3a85ea2e84521c9583b7bcfa68490bc9/src/commands/install.rs#L36

Or, more likely it's an issue with the attributes where the enum is declared
https://github.com/devklick/gdman-rs/blob/a3d3ac0e3a85ea2e84521c9583b7bcfa68490bc9/src/common.rs#L8

**Comment by devklick** [comment]
Note that when running `gdman install --help`, it shows us conflicting information:
```
  -p, --platform <PLATFORM>          Specifies the target platform [default: mac-os] [possible values: windows, linux, macos]
```
The default is `mac-os`, but the allowed values are `windows, linux, macos`.

**Comment by devklick** [comment]
Fixed in 0.0.7
